engine/apps/api/serializers/alert_group.py

```diff
@@ -242,8 +242,6 @@ def get_limited_alerts(self, obj):
         alerts = obj.alerts.order_by("-pk")[:100]
         return AlertSerializer(alerts, many=True).data
 
-    @extend_schema_field(UserShortSerializer(many=True))
+    @extend_schema_field(PagedUserSerializer(many=True))
```

Comment:
this change basically just adds the "important" field on these user objects (necessary for the new mockups/UX)